src/overlays/actors/ovl_Obj_Tokei_Tobira/z_obj_tokei_tobira.c

```diff
@@ -36,16 +37,150 @@ static InitChainEntry D_80ABD750[] = {
     ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
 };
 
-#endif
+Vec3f D_80ABD760 = { 0.0f, 0.0f, 80.0f };
+s16 D_80ABD76C[2] = { 0xC000, 0x4000 };
+s32 D_80ABD770[2] = { 0x060012B0, 0x06001590 };
+f32 D_80ABD778[2] = { 1.0f, -1.0f };
+Gfx *D_80ABD780[4] = { (Gfx *)0x06001108, (Gfx *)0x060013E8, NULL, NULL };
+
+
+//TODO: Deal with the data
+
+/*extern InitChainEntry D_80ABD750;
+extern Vec3f D_80ABD760;
+extern s16 D_80ABD76C[];
+extern s32 D_80ABD770;*/
+
+/**
+ *  Object's Initialization
+ */
+void ObjTokeiTobira_Init(Actor* thisx, PlayState* play) {
+    ObjTokeiTobira* this = THIS;
+    s32 pad;
+    s32 params = this->dyna.actor.params & 1;
+    Vec3f objPos; // Object's position
```

Comment:
As I stated in my other review, this comment is redundant and it should be removed.
Also, this temp can be simply named  `pos`